core/tests/protractor_desktop/wipeout.js

```diff
@@ -18,6 +18,7 @@
 
 var general = require('../protractor_utils/general.js');
 var users = require('../protractor_utils/users.js');
+var waitFor = require('../protractor_utils/waitFor.js');
```

Comment:
Interesting, how did this work earlier?

Reply:
Ah, I had fixed this before and then someone else fixed it, so now there's a merge issue. I'll repush, thanks!